src/CoreBundle/Serializer/BaseSerializerHandler.php

```diff
@@ -13,24 +13,26 @@
 
 namespace Sonata\CoreBundle\Serializer;
 
-if (!class_exists(\Sonata\Serializer\BaseSerializerHandler::class, false)) {
+use Sonata\CoreBundle\Serializer\SerializerHandlerInterface as LegacySerializerHandlerInterface;
+
+if (!class_exists(\Sonata\Form\Serializer\BaseSerializerHandler::class, false)) {
     @trigger_error(
-        'The '.__NAMESPACE__.'\BaseSerializerHandler class is deprecated since version 3.13.0 and will be removed in 4.0.'
-        .' Use Sonata\Serializer\BaseSerializerHandler instead.',
+        'The '.__NAMESPACE__.'\BaseSerializerHandler class is deprecated since version 3.19.0 and will be removed in 4.0.'
```

Comment:
```suggestion
        'The '.__NAMESPACE__.'\BaseSerializerHandler class is deprecated since sonata-project/core-bundle 3.19.0 and will be removed in version 4.0.'
```